### PR TITLE
chore(RHTAPSRE-437): Adding Control Plane ArgoCD Dashboard

### PR DESCRIPTION
**Pull request by manish-jangra**
This will add a `configmap` to be used in app-interface configuration to create a Grafana dashboard.

### CODE REVIEW

File: dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,928 @@
+apiVersion: v1
+data:
+  grafana-dashboard.json: |
```
Comment (avi-biton) [inline dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml:3]: I suggest renaming it to something to reflect the dashboard, maybe `konflux-argocd-dashboard.json`

File: dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,928 @@
...
+      },
+      "timezone": "",
+      "title": "Konflux ArgoCD",
+      "uid": "OFb_0IASz",
+      "version": 1,
+      "weekStart": ""
+    }
+kind: ConfigMap
+metadata:
+  name: grafana-dashboard-stonesoup-argocd
```
[the remark this answers is not in the document]
Comment (avi-biton) [reply dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml:924]: I think we should also rename the file:
`grafana-dashboard-stonesoup-argocd.configmap.yaml` -> `grafana-dashboard-konflux-argocd.configmap.yaml`

File: dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml (hunk 1 above)
Comment (manish-jangra) [reply dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml]: changed

File: dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (manish-jangra) [reply dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml]: changed

**Comment by avi-biton** [comment]
lgtm

File: dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,928 @@
...
+            "current": {
+              "selected": false,
+              "text": "appsres05ue1-prometheus",
+              "value": "appsres05ue1-prometheus"
+            },
+            "hide": 0,
+            "includeAll": false,
+            "label": "Prometheus",
+            "multi": false,
+            "name": "Source",
```
Comment (avi-biton) [inline dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml:815]:
```suggestion
            "name": "datasource",
```

Comment (manish-jangra) [reply dashboards/argocd/grafana-dashboard-stonesoup-argocd.configmap.yaml]: fixed

**Pull request merged**